### ISSUE TITLE
Update to latest Angular

### ISSUE THREAD

**Issue by KevinCorcor** [irustm/angular-nodegui#16]
cc: @irustm 
Approaching a year since the last commit on this repo. 

Is there any plans to update its dependencies? What are the blockers if any? 
I have had a look at updating them myself but I am new to angular and nodegui so I am finding it slow to reverse engineer things that break.

**Comment by irustm** [comment]
Hi! Thank you for your interest in this project!

Now nodegui itself has stopped developing, so I do not see the point in supporting this project. 
If you have other information please let me know about it.

And if you have any other questions, happy to help.

**Comment by KevinCorcor** [comment]
Agh I see. I hadn't realised this. Pity, it is a very interesting and useful framework. Its probably best I don't build an app on a framework with little future plans despite the current benefits. Thanks for the reply and if I come back around to it with questions I will let you know. Thanks!

**Comment by HealMah** [comment]
> Hi! Thank you for your interest in this project!
> 
> Now nodegui itself has stopped developing, so I do not see the point in supporting this project. If you have other information please let me know about it.
> 
> And if you have any other questions, happy to help.

No, actually it's not abandoned. Still updating even in 2024. So please do update the dependencies.
